### PR TITLE
Edit in place missing checkboxes and text

### LINKED ISSUE #7487: Missing Edit in place documentation
## Description
The Edit in place checkbox and text is missing from these tools in the vector geometry section: Affine, Covert To Curved Geometries, Extract Specific Vertices, Extract Vertices, Force Right Hand Rule, Geodesic Line Split at antimeridian and, Split by max length. I first noticed that the Convert to Curve was missing and then double check them by looking at the tools that were available when I had edit in place selected with a point, line, and polygon layer. Some of them such as Extract Vertices I can't actually get to work because they require a line or polygon input but only work as edit in place on point geometries. But this is also true of some others such as Centroids that are already tagged as edit in place so maybe I'm missing something. 

I have made the changes and have a pull request ready but wasn't sure if it is best practice to add an issue first. If anyone has any guidance on that let me know. 

Thanks   

Page URL: https://docs.qgis.org/testing/en/docs/user_manual/processing_algs/qgis/vectorgeometry.html

### PR DESCRIPTION
**Pull request by baswein**
Goal: Add Missing Edit in place documentation for Affine, Covert To Curved Geometries, Extract Specific Vertices, Extract Vertices, Force Right Hand Rule, Geodesic Line Split at antimeridian and, Split by max length.

Ticket(s): fix #7487 

- [x] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by baswein** [comment]
@DelazJ - I rebased the branch and it looks like it worked but I don't really know enough to be able to tell. It also looks like the Doctest build is failing

**Review by DelazJ — COMMENTED** [review]
Thanks @baswein 
I don't think we need to differentiate multi/single geometry type. I'd just keep point, line and polygon because (and I didn't check all) but the edit-in-place for single geometry is still allowed. Whether the output is different or not is not the question IMHO.
For me, these terms implicitly cover both single and multi shapes contexts, and let's not complicate (e.g. one can ask whether mentioning `polygon` for bounding boxes means that edit-in-place is not allowed for `multipolygon`)

File: docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -2234,6 +2254,10 @@ the new value.
 If no Z values exist, the geometry will be upgraded to include the Z
 dimension.
 
+|checkbox| Allows
+:ref:`features in-place modification <processing_inplace_edit>`
+of point, line, and polygon features with Z enabled
```
[the remark this answers is not in the document]
Comment (baswein) [reply docs/user_manual/processing_algs/qgis/vectorgeometry.rst:2254]: Correct- If the layer does not support Z values the algorithm can't be used with edit in place won't show up when edit in place is activated.

File: docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 2]
```diff
@@ -694,6 +705,7 @@ as for the original features.
 
 |checkbox| Allows
 :ref:`features in-place modification <processing_inplace_edit>`
+of multipoint features
```
[the remark this answers is not in the document]
Comment (baswein) [reply docs/user_manual/processing_algs/qgis/vectorgeometry.rst:703]: Sounds good - who am I to stand in the way of useless but possible. 😄

**Comment by baswein** [comment]
@DelazJ Thank you so much for walking me through the mess I got myself in. I did all the steps you suggested. And it seemed to work.

**Pull request merged**